libs/service-portal/education-license/src/screens/EducationLicense/components/LicenseCards/LicenseCards.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+import { gql, useQuery } from '@apollo/client'
+import { Query } from '@island.is/api/schema'
+import { Box, Button } from '@island.is/island-ui/core'
+import { EducationCard } from '@island.is/service-portal/core'
+
+const educationLicenseQuery = gql`
+  query educationLicenseQuery {
+    educationLicense {
+      id
+      school
+      programme
+      date
+    }
+  }
+`
+
+const LicenseCards = () => {
+  const { data } = useQuery<Query>(educationLicenseQuery)
+  return (
+    <>
+      {data?.educationLicense.map((license, index) => (
```

Comment:
You'll most likely get `map of undefined` error here.

Don't need the react fragment as well.

Aaaand since I'm commenting :D, the naming conventions should be:
```graphql
const EducationLicenseQuery = gql`
  query EducationLicenseQuery {
    educationLicense {
      ...
```